buildenv/jenkins/openjdk_tests

```diff
@@ -235,9 +237,24 @@ timestamps{
                             }
                         }
                         if (!isNodeIdle) {
-                            println "Cannot find any idle nodes. Starting dynamic vm"
-                            LABEL = LABEL.minus("ci.role.test&&")
-                            LABEL += '&&ci.agent.dynamic'
+                            if (params.CLOUD_PROVIDER != null && params.CLOUD_PROVIDER in dynamicAgents) {
+                                dynamicLabel = params.CLOUD_PROVIDER
+                            } else if (dynamicAgents.size() >= 1) {
```

Comment:
I believe this is the case (1) --> 7 as I mentioned in https://github.com/adoptium/aqa-tests/pull/4307#issuecomment-1470272296